### ISSUE TITLE
Which VS version should I consider when installing mod_wsgi on Windows?

### ISSUE THREAD

**Issue by alexvelickiy** [GrahamDumpleton/mod_wsgi#212]
Hi,

I have Python 3.6 installed on Windows 8.1 as well as Microsoft Visual C++ Build Tools 2015 (x64)
When start python interactive interpreter, it displays: 

`Python 3.6.2 (v3.6.2:5fd33b5, Jul  8 2017, 04:57:36) [MSC v.1900 64 bit (AMD64)] on win32`

My Visual Studio successfully compiled some packages for me.

But when I try to install mod_wsgi with pip (pip install mod_wsgi), I get Visual studio error:

`error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\link.exe' failed with exit status 1120`

I assume that it may be due to Visual Studio version that apache2 was built with. Mine is:

```
Server version: Apache/2.4.10 (Win32)
Apache Lounge VC11 Server built:   Jul 17 2014 11:50:08
```

If this is the case, why do python tries to compile mod_wsgi with apache's Visual Studio version?

Also, if I decide to download precompiled version of mod_wsgi from http://www.lfd.uci.edu/~gohlke/pythonlibs/#mod_wsgi, which VS version should I relate to: apache's one, or the one used by Python? And what is cp34, cp35, cp36 there?

**Comment by GrahamDumpleton** [comment]
You need to supply the full set of error messages. That is only the final status code and doesn't show what the problem is.

At a guess I would say the problem is that you don't have Apache installed where it is expected to be, or you are using an Apache distribution which doesn't provide header files and so can't be used when compiling third party Apache modules. I only recommend the Apache from:

* https://www.apachelounge.com/

You are better off compiling from source code than using those pre-compiled binaries as that guarantees better it matches what you have.

**Comment by alexvelickiy** [comment]
Sorry, cannot supply it as I already successfully installed mod_wsgi. I solved it by downloading VC14 Apache 2.4 binaries from apachelounge (https://www.apachelounge.com/download/VC14/). Apache 2.4.27 Win64 in my case to be exact.

And set environmental variable that mod_wsgi installation asked to location of this newly downloaded Apache. And it worked.
